### ISSUE TITLE
Docker image does not contain dgraphzero binary

### ISSUE THREAD

**Issue by danielsmithsd** [dgraph-io/dgraph#1611]
```
Daniels-MacBook-Pro:~ dsmith$ docker run -it dgraph/dgraph bash
root@0699edccea1b:/dgraph# ls -l /usr/local/bin/
total 39720
-rwxrwxr-x 1 root root 13179256 Oct  9 04:58 bulkloader
-rwxrwxr-x 1 root root 15424600 Oct  9 04:58 dgraph
-rwxrwxr-x 1 root root 12065144 Oct  9 04:58 dgraphloader
root@0699edccea1b:/dgraph#
root@0699edccea1b:/dgraph# find / -name dgraphzero
root@0699edccea1b:/dgraph#
```

**Comment by lazharichir** [comment]
Have you run it like explained here: https://docs.dgraph.io/get-started/#from-docker-image

?

**Comment by danielsmithsd** [comment]
Yep, and this started me down the road:
```
Daniels-MacBook-Pro:~ dsmith$ docker run -it -p 8080:8080 -p 9080:9080 -v ~/dgraph:/dgraph --name dgraph dgraph/dgraph dgraphzero -w zw
docker: Error response from daemon: oci runtime error: container_linux.go:265: starting container process caused "exec: \"dgraphzero\": executable file not found in $PATH".
```

**Comment by pawanrawal** [comment]
Hi @danielsmithsd 

Thanks for reporting this. Our CI system builds the docker images. Yesterday after the `v0.8.3` release a fix was pushed to the previous release(v0.8.2) 761195a0d99062b09ec6cde13b962bc51bbf2303 which lead to overwriting of the latest docker image (which now pointed to the v0.8.2).

I have pushed the latest image. Can you try again?

**Comment by pawanrawal** [comment]
We should also fix the script https://github.com/dgraph-io/dgraph/blob/master/contrib/nightly/build.sh#L222 to get the latest version from Github before deciding whether to tag the current version as latest or not.

**Comment by danielsmithsd** [comment]
@pawanrawal that fixed it up.  Thanks!

**Comment by pawanrawal** [comment]
A fix for the issue mentioned was merged to `release/v0.8.3` and `master`.
